app/controllers/solidus_auth_devise/controller_helpers/deprecated_routes.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module SolidusAuthDevise
+  module ControllerHelpers
+    module DeprecatedRoutes
+      extend ActiveSupport::Concern
+
+      included do
+        before_action :deprecates_solidus_auth_devise_routes
+      end
+
+      private
+
+      def deprecates_solidus_auth_devise_routes
+        if params[:deprecated_route]
+          Spree::Deprecation.warn(
+            <<~TEXT
+              This route is deprecated: #{request.fullpath.inspect}.
+              It will be removed in solidus_auth_devise v3.
+              If you want to continue using this route please define it in your application code:
+
+              Spree::Core::Engine.routes.draw do
```

Comment:
This works only for the frontend right? For backend routes it will also have the `namespace :admin do` part. Not that we need it now, just wanted to be sure.

Reply:
For backend we already `skip: :all` so it should be already streamlined, luckily 🍀

Reply:
Can't find this `skip: app` for backend routes, can you please point it here?

Reply:
here https://github.com/nebulab/solidus_auth_devise/blob/elia%2Bkennyadsl/fix-routes/config/routes.rb#L75

Reply:
Ah, got what you mean now. Still, the `with_options deprecated_route: true do` could be used in the future on a backend route. I'm just saying that if that happens, this concern will be printing a non-accurate message, because it only works with frontend routes.

Reply:
That's true, but I trust we'll remove everything when releasing v3.0